### DISCUSSION TITLE
How to layout Text inside a button

### DISCUSSION THREAD

**Discussion by mjehrhart** [emilk/egui#1195] [Q&A]
Below, I have a panel filled with buttons. Is there a way to place and align multiple labels with in the button?

In this screenshot, I want to align all the "x dupes" together. I don't know if this is possible or not.

![Screen Shot 2022-02-01 at 3 52 53 PM](https://user-images.githubusercontent.com/97703291/152071404-e40e978f-6e9c-4067-a981-7882e57757a2.png)

Thanks.

**Comment by emilk** [comment]
That's not currently possible, unless you switch to a monospace font and align the characters

**Reply by mjehrhart** [reply]
Thanks for the info. I ended up formatting the strings themselves, so all ends well.
